twisted/conch/scripts/ckeygen.py

```diff
@@ -113,8 +113,7 @@ def generateDSAkey(options):
 
 
 def printFingerprint(options):
-    if not options.has_key('format'):
-        options['format'] = 'md5-hex'
+    options.setdefault('format', 'md5-hex')
```

Comment:
this should no longer be needed as the option parsing code should take care of inserting the default value